### DISCUSSION TITLE
FF7 7th Heaven crashing

### DISCUSSION THREAD

**Discussion by RoadlBlock** [julianxhokaxhiu/FFNx#567] [Q&A]
Anybody experiencing crash on 7th heaven? I tried deactivating all mods and it still crashes, I get a warning saying about FFNX. just happened 3 days ago. before that I playing with mods just fine. This happens when in Kalm doing a back story with Sephiroth on the truck then about to enter battle with the dragon
![FFNX](https://user-images.githubusercontent.com/132066757/235143621-77ecd590-6832-47a6-811f-558cfef3d4c6.png)

**Comment by eve-atum** [comment]
Head to the qhimm or tsunamods discord, and be ready to provide screenshots.  Sounds like you have a mod conflict of some sort, or a windows protection setting is being too restrictive.
